src/NServiceBus.Testing.Fakes/NServiceBus.Testing.Fakes.csproj

```diff
@@ -9,26 +9,5 @@
 
   <ItemGroup>
     <ProjectReference Include="..\NServiceBus.Core\NServiceBus.Core.csproj" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Particular.Packaging" Version="0.2.1" PrivateAssets="All" />
-  </ItemGroup>
-
-  <PropertyGroup>
-    <PackageId>NServiceBus.Testing.Fakes.Sources</PackageId>
-    <Description>Unit testing fakes and helpers for NServiceBus core functionality</Description>
-    <IncludeBuildOutput>false</IncludeBuildOutput>
-  </PropertyGroup>
-
-  <!-- Workaround for https://github.com/dotnet/sdk/issues/1479 -->
-  <ItemGroup>
-    <_PackageFiles Include="**\*.cs">
-      <BuildAction>Compile</BuildAction>
-      <PackagePath>content\App_Packages\NSB.Testing.Fakes\;contentFiles\cs\any\NSB.Testing.Fakes\</PackagePath>
-    </_PackageFiles>
-    <_PackageFiles Remove="**\obj\**\*.cs" />
-  </ItemGroup>
-  <!-- End Workaround -->
-
+  </ItemGroup>  
```

Comment:
nit: Let's keep the empty line between the `</ItemGroup>`  and the `</Project>`.